pkg/tfvars/baremetal/baremetal.go

```diff
@@ -75,13 +82,15 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			"power_interface":      accessDetails.PowerInterface(),
 			"raid_interface":       accessDetails.RAIDInterface(),
 			"vendor_interface":     accessDetails.VendorInterface(),
+			"instance_info": map[string]interface{}{
+				"deploy_boot_mode": bootMode,
```

Comment:
Can't we use the helper methods instead from the new baremetal-operator PR?

Reply:
I thought we were going to take the version of the baremetal-operator change that did not include the method asking the BMC for the instance details? Even if we did take them, the method only provides a default boot mode. The caller still has to assemble the map to pass to gophercloud.

Reply:
Yea sorry I missed that this was no longer some helper method but in the provisioning code itself, so we'll have to duplicate it.

Reply:
If it ever becomes more complicated than what it is now, we can put it in a separate package in the BMO repo so we can reuse it.